### PR TITLE
Update to ember-cli@2.16 blueprint.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-/* global require, module */
+/* eslint-env node */
 var path = require('path');
-var EmberApp = require('ember-cli/lib/broccoli/ember-addon');
+'use strict';
```
Comment (Turbo87) [inline ember-cli-build.js]: I think this needs to happen before any other expression

Comment (rwjblue) [reply ember-cli-build.js]: yes, confirm

File: .travis.yml [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
-  - EMBER_TRY_SCENARIO=ember-release
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+  matrix:
+    # we recommend new addons test the current and previous LTS
+    # as well as latest stable release (bonus points to beta/canary)
+    - EMBER_TRY_SCENARIO=ember-lts-2.8
```
Comment (mixonic) [inline .travis.yml]: We should still go back to 1.13 IMO unless we're cutting a new major.

File: ember-cli-build.js (hunk 1 above)
Comment (mixonic) [inline ember-cli-build.js]: Yah, must be resolved before this can land.

**Comment by rwjblue** [comment]
Replaced by #57

**Pull request closed without merging**